### PR TITLE
Extend batch_update_users script to be able to update 'isArchived' user attribute

### PR DESCRIPTION
**Pull request by garethbrickman**

### CODE REVIEW

File: batch_update_users.py [hunk 1]
```diff
@@ -141,6 +141,15 @@ def main():
...
                 member_type = None
+        try:
+            if len(m['isArchived']) > 1 and m['isArchived'] == "True":
+                member_isArchived = True
+            elif len(m['isArchived']) > 1 and m['isArchived'] == "False":
+                member_isArchived = False
+            else:
+                member_isArchived = None
+        except KeyError:
+                member_isArchived = None
```
Comment (HiranmayaGundu) [inline batch_update_users.py:152]: Try this instead? 

```suggestion
            archived_value = m.get('isArchived', None)
            if archived_value == "True":
                member_isArchived = True
            elif archived_value == "False":
                member_isArchived = False
            else:
                member_isArchived = None
```

This way we don't rely on the error type to set `member_isArchived=None`

File: batch_update_users.py [hunk 2]
```diff
@@ -161,7 +170,10 @@ def main():
                     payload['email'] = member_new_email
             if member_type:
                 payload['memberType'] = member_type
-
+            if member_isArchived is True:
+                payload['isArchived'] = member_isArchived
+            elif member_isArchived is False:
+                payload['isArchived'] = member_isArchived
```
Comment (HiranmayaGundu) [inline batch_update_users.py:176]:
```suggestion
            if member_isArchived is not None :
                payload['isArchived'] = member_isArchived
```

**Review by HiranmayaGundu — COMMENTED** [review]
Couple of comments, but looks good to me otherwise

**Comment by garethbrickman** [comment]
@HiranmayaGundu Thanks! Committed your suggestions

**Review by HiranmayaGundu — APPROVED** [review]
🚀

**Pull request merged**